docs/releases/4.2.txt

```diff
@@ -26,6 +26,20 @@ and only officially support the latest release of each series.
 What's new in Django 4.2
 ========================
 
+Psycopg 3 support
+-----------------
+
+Django now supports `psycopg`_ version 3.1 or higher. To update your code,
+install the `psycopg library`_, you don't need to change the
+:setting:`ENGINE <DATABASE-ENGINE>` as ``django.db.backends.postgresql``
+supports both libraries.
+
+Support for ``psycopg2`` is aim to be deprecated and removed at some point in
+the future.
```

Comment:
`aimed` but... 

```suggestion
Support for ``psycopg2`` is likely to be deprecated and removed at some point in
the future.
```

Are we not saying `will be deprecated in 5.0 and removed in 6.0`?

Reply:
> Are we not saying `will be deprecated in 5.0 and removed in 6.0`?

Let's first check how stable it is and how many issue will be reported.

Reply:
But assuming all goes well yes? (I leave it to you, but I might be inclined to be more informative about the actual aim...)